### PR TITLE
Add powershell script to run tests in parallel in build

### PR DESCRIPTION
**Pull request by benjaminpetit**
Using background jobs to start xunit console, one per test assembly.

### CODE REVIEW

File: src/Parallel-Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+param(
+    [string[]] $assemblies,
+    [string] $testFilter,
+    [string] $outDir)
+
+$xunitRunner = $(Join-Path $PSScriptRoot 'packages\xunit.runner.console.2.1.0\tools\xunit.console.exe ')
```
Comment (ReubenBond) [inline src/Parallel-Tests.ps1]: It's a little long-winded, but if we want version tolerance & platform independence (join-path), then perhaps something like this?
```ps
$xunitRunner = (Get-Item $([System.IO.Path]::Combine($PSScriptRoot, "packages","xunit.runner.console.*","tools", "xunit.console.exe")) | Sort-Object FullName | Select-Object -Last 1).FullName
```

Maybe that's overkill & maybe there's a better way.

**Comment by jdom** [comment]
I'll fix the build error tomorrow

File: src/Parallel-Tests.ps1 (hunk 1 above)
Comment (benjaminpetit) [inline src/Parallel-Tests.ps1]: Totally overkill :) I think it should be easier to just have a if/else to choose the path when we will be cross platform. Maybe we will not use the same exec or the same version?

I like the idea to get the latest one, will see if I can come up with a quick solution

Comment (benjaminpetit) [inline src/Parallel-Tests.ps1]: Last commit should implement your suggestions!

**Comment by benjaminpetit** [comment]
@dotnet-bot test this please

**Comment by benjaminpetit** [comment]
@dotnet-bot test this please

**Pull request merged**